windows/ReactNativeWebView/ReactWebView2Manager.cpp

```diff
@@ -120,7 +126,7 @@ namespace winrt::ReactNativeWebView::implementation {
         winrt::IJSValueReader const& commandArgsReader) noexcept {
         auto control = view.as<winrt::ContentPresenter>();
         auto content = control.Content();
-        auto webView = content.as<winrt::WebView>();
+        auto webView = content.as<winrt::WebView2>();
```

Comment:
just to confirm, this function is only used for WV2 and not for WV1 anymore, right?

Reply:
Yes! Because this is the ViewManager for the WebView2 control.